### PR TITLE
convert package to PythonCall instead of PyCall

### PR DESCRIPTION
**Pull request by palday**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,55 +10,26 @@ Julia interface to MNE-Python via PyCall
...
-You can test your setup with `using PyCall; pyimport("mne")`.
+and PyMNE will attempt to install when the package is built: this should happen
+more or less automatically via [`CondaPkg`](https://github.com/cjdoris/CondaPkg.jl).
+You can configure various options via `CondaPkg`. MNE-Python is installed via
+Conda, not via pip.
 
 ## Usage
 
-In the same philosophy as PyCall, this allows for the transparent use of
+In the same philosophy as Python, this allows for the transparent use of
```
Comment (ararslan) [inline README.md:23]: Did you mean PythonCall?

File: Project.toml [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
-PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
+CondaPkg = "992eb4ea-22a4-4c89-a5bb-47a3300528ab"
+PythonCall = "6099a3de-0909-46bc-b1f4-468b9a2dfc0d"
+Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 
 [compat]
-PyCall = "1.90"
-julia = "1"
+CondaPkg = "0.2.11"
+PythonCall = "0.9.4"
```
Comment (ararslan) [inline Project.toml:13]:
```suggestion
PythonCall = "0.9.4"
Reexport = "1"
```

File: format/Project.toml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[deps]
+JuliaFormatter = "98e50ef6-434e-11e9-1051-2b60c6c9e899"
```
Comment (ararslan) [inline format/Project.toml:2]:
```suggestion
JuliaFormatter = "98e50ef6-434e-11e9-1051-2b60c6c9e899"

[compat]
JuliaFormatter = "1"
```

File: test/runtests.jl [hunk 4]
```diff
@@ -1,22 +1,25 @@
...
-    @test raw.get_data() == dat
-    @test get_info(raw) isa PyObject
-    @test raw.info isa Dict
-    @test_throws PyError PyMNE.io.RawArray(dat, naive_info)
+    dat = rand(1, 100)
+    info = PyMNE.create_info(pylist(["a"]), 100)
+    @test info isa Py
+    @test pyconvert(String, only(info.ch_names)) == "a"
+    # XXX why all the pyconvert? well comparisons of python objects give you
+    # a python boolean, so we need to explicitly convert for @OTDE
```
Comment (ararslan) [inline test/runtests.jl:11]: Why for Seth?

Comment (palday) [reply test/runtests.jl:11]: wat. It looks like autocomplete did something weird

Comment (palday) [reply test/runtests.jl:11]:
```suggestion
    # a python boolean, so we need to explicitly convert for `@test`
```

File: README.md (hunk 1 above)
Comment (palday) [reply README.md:23]:
```suggestion
In the same philosophy as PythonCall, this allows for the transparent use of
```

File: src/PyMNE.jl [hunk 5]
```diff
@@ -4,30 +4,20 @@ module PyMNE
...
-# TODO: examine how to do wrappers for subsubmodules. for now it's not a huge deal
-#       because the wrapper just gets put in the top-level package namespace and so
-#       is still accessible
-
-include("wrappers.jl")
+const mne = PythonCall.pynew()
 
 function __init__()
     # all of this is __init__() so that it plays nice with precompilation
     # see https://github.com/JuliaPy/PyCall.jl/#using-pycall-from-julia-modules
```
Comment (ararslan) [inline src/PyMNE.jl:19]: Should we update the reference here since we aren't using PyCall?

Comment (palday) [reply src/PyMNE.jl:19]:
```suggestion
    # see https://github.com/cjdoris/PythonCall.jl/blob/5ea63f13c291ed97a8bacad06400acb053829dd4/src/Py.jl#L85-L96
```

**Review by ararslan — APPROVED** [review]

**Pull request merged**